src/utilities/toast-manager.test.ts

```diff
@@ -1,3 +1,92 @@
+import faker from "faker";
+import { ToastManager } from "./toast-manager";
+
 describe("ToastManager", () => {
-    test.skip("TODO: Backfill tests for issue https://github.com/AndcultureCode/AndcultureCode.JavaScript.React.Components/issues/34", () => {});
+    test("when error toast created, returns toastId", () => {
+        // Arrange
+        const testToastId = faker.random.number(99);
+        const toastContent = faker.random.word();
+        const toastOptions = { toastId: testToastId };
+
+        // Act
+        const toastId = ToastManager.error(toastContent, toastOptions);
+
+        // Assert
+        expect(toastId).toBe(testToastId);
+    });
+
+    test("when info toast created, returns toastId", () => {
+        // Arrange
+        const testToastId = faker.random.number(99);
+        const toastContent = faker.random.word();
+        const toastOptions = { toastId: testToastId };
+
+        // Act
+        const toastId = ToastManager.info(toastContent, toastOptions);
+
+        // Assert
+        expect(toastId).toBe(testToastId);
+    });
+
+    test("when success toast created, returns toastId", () => {
+        // Arrange
+        const testToastId = faker.random.number(99);
+        const toastContent = faker.random.word();
+        const toastOptions = { toastId: testToastId };
+
+        // Act
+        const toastId = ToastManager.success(toastContent, toastOptions);
+
+        // Assert
+        expect(toastId).toBe(testToastId);
+    });
+
+    test("when warn toast created, returns toastId", () => {
+        // Arrange
+        const testToastId = faker.random.number(99);
+        const toastContent = faker.random.word();
+        const toastOptions = { toastId: testToastId };
+
+        // Act
+        const toastId = ToastManager.warn(toastContent, toastOptions);
+
+        // Assert
+        expect(toastId).toBe(testToastId);
+    });
+
+    test("when toast dismissed, calls dismiss method", () => {
+        // Arrange
+        const dismissMethodSpy = jest.spyOn(ToastManager, "dismiss");
+        const testId = faker.random.number(99);
+
+        // Act
+        ToastManager.dismiss(testId);
+
+        // Assert
+        expect(dismissMethodSpy).toHaveBeenCalled();
+    });
+
+    test("when dismiss All toasts, calls dismissAll method", () => {
```

Comment:
The suggested region/describe blocks would probably help with this test name

Reply:
@brandongregoryscott - Thanks for your feedback. Can you take a second look when you have time.